requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy==1.24.3
+opencv_python==4.7.0.72
+Pillow==9.5.0
+PySimpleGUI==4.60.4
+PyYAML==6.0
+segment_anything==1.0
+torch==1.12.1+cu113
```

Comment:
bu kisim device'taki gpu'ya gore degiskenlik gosterebilir, o sebeple bunu cikartmak durumunda kalabiliriz.

Reply:
tabi doğru diyorsun abi. pipreqs ile yaptığım için otomatik oluştu. onu sadece torch diye düzeltebilirim.

Reply:
Tamamdir, ben Dockerfile olusturuyorum. pytorch installation'i onun icine koyabiliriz